nexus/Cargo.toml

```diff
@@ -48,9 +49,14 @@ path = "../api_identity"
 version = "0.4"
 features = [ "serde" ]
 
+[dependencies.diesel-dtrace]
+git = "https://github.com/oxidecomputer/diesel-dtrace"
+branch = "main"
```

Comment:
I went through and took a look at the new repo. Couple quick notes:

- Looks good! I'm pleasantly surprised we could pull this out so cleanly into a crate of its own.
- Are the probes all prefixed by `diesel` or `diesel_db`? I'm seeing examples referencing `diesel*:::`, but the provider looks like it's called `diesel_db`.

Reply:
The probe is actually `diesel_db`. I'm not sure that's the right name TBH, since the `diesel` bit is almost irrelevant. Let me know if you've got suggestions.

Reply:
Ehh definitely feels like those "pick a name and stick with it" sorta situations, just wanted to be clear on what that was. IMO either would be fine.